<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_login</class>
 <widget class="QDialog" name="admin_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>559</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>671</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LOGIN FORM</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_LOGIN">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_username">
       <property name="text">
        <string>USERNAME</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_username</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_username">
       <property name="maximumSize">
        <size>
         <width>112</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">
</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::LogicalMoveStyle</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>181</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_password">
       <property name="text">
        <string>PASSWORD</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_2</cstring>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="maximumSize">
        <size>
         <width>112</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_eye">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>85</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/resources/eye_off.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>421</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft PhagsPa</family>
     <pointsize>15</pointsize>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>HOME RENTAL MANAGEMENT</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DIDN'T SIGNUP YET?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SIGNUP">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>460</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>SIGNUP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LOGIN FORM</string>
   </property>
  </widget>
  <action name="actionico">
   <property name="icon">
    <iconset resource="RESOURCE.qrc">
     <normaloff>:/resources/resources/house_rental_icon.ico</normaloff>:/resources/resources/house_rental_icon.ico</iconset>
   </property>
   <property name="text">
    <string>ico</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="RESOURCE.qrc"/>
 </resources>
 <connections/>
</ui>
